gateway/middleware.go

```diff
@@ -36,7 +36,7 @@ func (a Authentication) Wrap(next http.Handler) http.Handler {
 		if ok {
 			next.ServeHTTP(sr, r)
 		} else {
-			level.Debug(logger).Log("msg", "No valid tenant credentials are found")
+			logrus.Infof("No valid tenant credentials are found")
```

Comment:
This will fill up the logs quickly for any service on the Internet. Let's keep it on debug